src/Stripe.net/Services/PaymentIntents/PaymentIntentSharedOptions.cs

```diff
@@ -27,15 +27,18 @@ public class PaymentIntentSharedOptions : BaseOptions
         [JsonProperty("on_behalf_of")]
         public string OnBehalfOf { get; set; }
 
-        [JsonProperty("payment_method_types")]
-        public List<string> PaymentMethodTypes { get; set; }
-
         [JsonProperty("receipt_email")]
         public string ReceiptEmail { get; set; }
 
+        [JsonProperty("payment_method_types")]
+        public List<string> PaymentMethodTypes { get; set; }
```

Comment:
Looks like this one was already in the correct alphabetical order, can you move it back?